src/library_syscall.js

```diff
@@ -710,6 +710,9 @@ var SyscallsLibrary = {
       });
     });
 #else
+    if (stream.stream_ops.fsync) {
+        return -stream.stream_ops.fsync(stream);
```

Comment:
indentation looks wrong here - should be 2 spaces, not 4.

We need to fix CI for this PR anyhow, can fix the indentation

Reply:
Very strange.    I'll guess I'll revert.  @abujalski can you re-upload a new PR?

Reply:
Reverted in 771fd6b27